### DISCUSSION TITLE
FofaViewer 批量查询 功能

### DISCUSSION THREAD

**Discussion by dointisme** [wgpsec/fofa_viewer#62] [Ideas]
建议添加full功能，只能查询一年的在某些应用场景不够适用。

**Comment by f1ashine** [comment]
可以具体描述下吗，看不懂

**Comment by 9018** [comment]
> 可以具体描述下吗，看不懂

full
boolean
默认为false，是否搜索全部数据，默认和页面搜索一样,只能搜索一年内的数据，指定为true搜索全部数据

**Comment by 9018** [comment]
> 可以具体描述下吗，看不懂

f1ashine 老师你好，作为甲方确实有时候领导会要求提交ip的归属地和发现的时间，期待会有这个功能^^
